lib/oca-epak/epak/client.rb

```diff
@@ -100,22 +177,30 @@ def get_operation_codes
       # @param [String] "From date" in DD-MM-YYYY format
       # @param [String] "To date" in DD-MM-YYYY format
       # @return [Array, nil] Contains an array of hashes with NroProducto and NumeroEnvio
-      def list_shipments(cuit, from_date, to_date)
+      def list_shipments(opts = {})
```

Comment:
@cavi21 Ok, I like this change. I'll have to keep in mind that after we merge this, we will need to bump the version to 2.0